__main__.py

```diff
@@ -16,7 +16,8 @@
 os.chdir(dirname)
 
 try:
-    os.system("touch Cached_Data.txt")
+    with open("Cached_Data.txt","w") as file:
+        file.close()
```

Comment:
Calling `close()` isn't required with using open as a context manager. It's called during the __exit__ method.